spec/requests/api/v1/people_spec.rb

```diff
@@ -0,0 +1,229 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'api/v1/people', type: :request do
+  describe 'GET /api/v1/people' do
+    it 'returns http status success' do
+      get api_v1_people_path
+      expect(response).to have_http_status(:success)
+    end
+
+    let(:people) do
+      [
+        { name: 'Isaac Newton', cpf: '000.001.003-05', birthday: '1943-01-04' },
+        { name: 'Leonhard Euler', cpf: '271.828.182-84', birthday: '1707-04-15' },
+        { name: 'Felipe', cpf: '000.000.000', birthday: '1900-01-01' }
```

Comment:
Esse cara pode ser o input do bulk insert. 🤔

Reply:
Eu tive problemas tentando usar o bulk insert ActiveRecord::ConnectionAdapters::SQLite3Adapter does not support skipping duplicates

Reply:
Decidimos ignorar o bulk. A versão do SQLITE que eu estou usando não permite usar o insert_all.